### PR TITLE
Housekeeping: refactoring, cleanup, updates

### PR DESCRIPTION
**Pull request by tilsche**
- update to metricq 5.0
- strict typing
- simplification of commands using new metricq_command

### CODE REVIEW

File: metricq_tools/summary.py [hunk 1]
```diff
@@ -66,28 +66,30 @@ def __init__(
...
+    async def add_data(
+        self, metric: str, timestamp: metricq.Timestamp, value: float
+    ) -> None:
         if self.print_data:
             click.echo(click.style("{}: {}".format(timestamp, value), fg="bright_blue"))
 
         self.timestamps[metric].append(timestamp.posix)
-        if self.last_timestamp[metric]:
-            self.intervals[metric].append(timestamp.posix - self.last_timestamp[metric])
+        if last_timestamp := self.last_timestamp[metric] is not None:
```
Comment (bmario) [inline metricq_tools/summary.py:87]: I'd rather have parentheses for the walrus.

File: metricq_tools/send.py [hunk 2]
```diff
@@ -26,40 +26,31 @@
...
         await self.send(self.metric, time=self.timestamp, value=self.value)
         await self.stop()
 
 
-@click.command()
-@click_log.simple_verbosity_option(logger, default="warning")
-@click.version_option(version=client_version)
-@metricq_server_option()
-@metricq_token_option(default="source-send")
+@metricq_command("source-tool-send")
```
Comment (bmario) [inline metricq_tools/send.py:53]:
```suggestion
@metricq_command(default_token="source-tool-send")
```

File: metricq_tools/check.py [hunk 3]
```diff
@@ -52,14 +46,12 @@ async def check_metric(metric):
         print(metric, aggregate)
 
 
-@click.command()
-@click_log.simple_verbosity_option(logger, default="warning")
-@metricq_server_option()
-def main(server):
-    """Check metrics for non-finite values."""
+@metricq_command("history-tool-check")
```
Comment (bmario) [inline metricq_tools/check.py:49]:
```suggestion
@metricq_command(default_token="history-tool-check")
```

**Comment by tilsche** [comment]
@bmario all comments addressed

**Review by bmario — APPROVED** [review]

**Pull request merged**